### DISCUSSION TITLE
How to override table.aliasName?

### DISCUSSION THREAD

**Discussion by rizalahmaddd** [simolus3/drift#2250] [Q&A]
Hi, is it possible to override  aliasName like actualName on table? i want to set my actualTable name = 'hr.work_work_location' with aliasName = 'workLocation' 

<img width="391" alt="Screenshot 2023-01-01 at 4 03 59 PM" src="https://user-images.githubusercontent.com/93845700/210166057-3b9a1364-8a2a-46f3-bbfe-f7c9ae00f4bb.png">

<img width="539" alt="Screenshot 2023-01-01 at 4 04 26 PM" src="https://user-images.githubusercontent.com/93845700/210165996-a25dedea-c373-40c2-9953-dc77f13bc82e.png">

thankyout

**Comment by simolus3** [comment]
> is it possible to override aliasName like actualName on table

The purpose of aliases is to [be able to use the same table in a query more than once](https://drift.simonbinder.eu/api/drift/databaseconnectionuser/alias), which can be useful in some instances. So you can use the `alias()` method to get an instance of your table with the `aliasedName` overridden, is that what you were looking for?
